gobblin-service/src/main/java/org/apache/gobblin/service/modules/utils/FlowCompilationValidationHelper.java

```diff
@@ -86,10 +86,16 @@ public Optional<Dag<JobExecutionPlan>> createExecutionPlanIfValid(FlowSpec flowS
     Map<String, String> flowMetadata = TimingEventUtils.getFlowMetadata(flowSpec);
 
     if (!jobExecutionPlanDagOptional.isPresent()) {
+      log.warn("No dag execution plan created for flowGroup: {} flowName: {}",
+          flowMetadata.get(TimingEvent.FlowEventConstants.FLOW_GROUP_FIELD),
+          flowMetadata.get(TimingEvent.FlowEventConstants.FLOW_NAME_FIELD));
```

Comment:
if we do anything other than logging the value, I recommend a default (even when it *should* always be set).  for logging only, it's safe enough to print `null`, if that's what's actually returned.